### PR TITLE
[#28] Removing multi author support from parsing single author fields.

### PR DESCRIPTION
**Pull request by richardhallett**
The reason we are doing this is it next to impossible to parse if it is actually a multiple author. Sometimes it might just be extra information about one author.
Ideally the author should just be "Richard Hallett" and then subsequent authors are added into the metadata as separate entries.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    bolognese (0.9.64)
```
Comment (kjgarza) [inline Gemfile.lock]: please separate commits for gem updates. 
Merry xmas

File: spec/writers/codemeta_writer_spec.rb [hunk 2]
```diff
@@ -26,8 +26,8 @@
       expect(json["@id"]).to eq("https://doi.org/10.5063/f1m61h5x")
       expect(json["@type"]).to eq("SoftwareSourceCode")
       expect(json["identifier"]).to eq("https://doi.org/10.5063/f1m61h5x")
-      expect(json["agents"].length).to eq(9)
-      expect(json["agents"].first).to eq("type"=>"Person", "name"=>"Matthew B. Jones", "givenName"=>"Matthew B.", "familyName"=>"Jones")
+      expect(json["agents"].length).to eq(1)
```
Comment (mfenner) [inline spec/writers/codemeta_writer_spec.rb]: May not be needed, as you check the whole hash in the next step.

File: lib/bolognese/author_utils.rb [hunk 3]
```diff
@@ -72,9 +64,6 @@ def cleanup_author(author)
       # remove spaces around hyphens
       author = author.gsub(" - ", "-")
 
-      # remove text in parentheses
-      author = author.sub(/\s*\(.+\)\s*/, '')
```
Comment (kjgarza) [inline lib/bolognese/author_utils.rb]: oh this blabal

File: lib/bolognese/author_utils.rb [hunk 4]
```diff
@@ -19,14 +19,6 @@ def get_one_author(author)
              parse_attributes(author.fetch("contributorName", nil)) ||
              author.fetch("name", nil)
 
-      if name.include?("; ")
```
Comment (kjgarza) [inline lib/bolognese/author_utils.rb]: I'm a bit worried about the removal of functionality but ok.

**Pull request merged**